### ISSUE TITLE
dataloader issue due to reading files with a name that is composed of mixed lower and upper case characters

### ISSUE THREAD

**Issue by vietvo89** [XenderLiu/MalConv-Pytorch#7]
Hi 

When I run your repo (training code), dataloader reads file name from a file that keeps file name and label as you instructed. But I got a problem with this when it cannot read a file name that is composed of mixed lower and upper case characters. I am not sure why it is the case. When I changed the file name to all lower cases or upper cases, it could read and loaded this file. But I could not change for all files since thousands of files have name with mixed lower and upper case characters. 

How can I modify your code to read all files with mixing upper and lower case characters? 

Thanks
